WordPress/gutenberg
# List view: Refactor ARIA attributes

## What?


This PR attempts to remove `aria-hidden="true"` from the links in the list view.

## Why?


This was a hack to get around block selection bugs in list view. I think I found a way to make it no longer necessary. There is a little extra screen reader verbosity, but I do not think this is a bad thing since the selection announcement will now take priority with `aria-live="assertive"`.

## How?


Changes multiple ARIA attributes and ensure `speak()` function gets assertive argument.

## Testing Instructions





1. Start a screen reader such as NVDA or Voiceover.
2. Open a post or page.
3. Insert different types of blocks and be sure to create inner blocks/child blocks.
4. Open the list view with alt+shift+o on Windows.
5. Notice how the screen reader still reliably reads the items.
6. Notice how the screen reader still announces expanded/collapsed states for inner blocks on the parent.
7. Notice how the default screen reader verbosity for links is restored.
8. Start selecting blocks. You should hear the number of blocks selected. Still may play around with the verbosity a bit.

### Testing Instructions for Keyboard


Provided above.

## Screenshots or screencast
